src/main.cc

```diff
@@ -300,7 +301,8 @@ inline void reset_optind() {
   optind = 0;
 #endif
 }
-
+const char * THEME_PRESETS_REPO_CLONING_URL = "https://github.com/Cetttok/testRepoForConkyThemes"; // it is example need to create norm repo
+const char * THEME_PRESETS_REPO_PATH = "/var/lib/conky/themes";
```

Comment:
Why are these in main.cc? You're declaring them here, just to pass them to the function in another file. Why not declare them above the function so they're easy to find and modify?

main.cc is already too large (much like conky.cc). Avoid placing more stuff in these two files.